src/components/Proposal/SentimentSurvey/SentimentSurvey.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react'
+
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+
+import { ReactionType, Survey, SurveyTopicAttributes, Topic } from '../../../entities/SurveyTopic/types'
+
+import './SentimentSurvey.css'
+import SentimentSurveyRow from './SentimentSurveyRow'
+
+interface Props {
+  survey: Survey
+  surveyTopics: Pick<SurveyTopicAttributes, 'label' | 'topic_id'>[] | null
+  isLoadingSurveyTopics: boolean
+  setSurvey: React.Dispatch<React.SetStateAction<Survey>>
+}
+
+const SentimentSurvey = ({ survey, surveyTopics, isLoadingSurveyTopics, setSurvey }: Props) => {
```

Comment:
Did you define the function as a variable for any special reason? I know it doesn't affect anything but it caught my attention 🤔

Reply:
No special reason. But we do it in several places, so it's not a special case either